### PR TITLE
feat: add option to hide suspicious user highlight

### PR DESCRIPTION
**Pull request by Niller2005**
#518

### CODE REVIEW

File: src/site/twitch.tv/modules/chat/ChatList.vue [hunk 1]
```diff
@@ -139,7 +140,7 @@ function onChatMessage(msg: ChatMessage, msgData: Twitch.AnyMessage, shouldRende
 		msg.setComponent(typeMap[0], { msgData: msgData });
 	}
 
-	if (msgData.type === MessageType.RESTRICTED_LOW_TRUST_USER_MESSAGE) {
+	if (msgData.type === MessageType.RESTRICTED_LOW_TRUST_USER_MESSAGE && showSuspiciousUser.value) {
```
Comment (MatinAniss) [inline src/site/twitch.tv/modules/chat/ChatList.vue:143]: Low Trust User ≠ Suspicious User

In my opinion their should be another setting to enable or disable low trust user highlights.

File: CHANGELOG-nightly.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 -   Links in chat messages now respect known TLDs instead of matching any url-like pattern
 -   Added an option to show timeouts/bans directly in the chat without being a moderator
+-   Add option to hide suspicious user highlight
```
Comment (lukashoracek) [inline CHANGELOG-nightly.md:7]: This should probably be "Added an option to hide suspicious user highlights" to match with the rest of the changelog

File: src/site/twitch.tv/modules/chat/ChatModule.vue [hunk 3]
```diff
@@ -353,6 +353,18 @@ export const config = [
...
+		label: "Show Suspicious User Highlights",
+		hint: "Whether or not to highlight users who are marked as a suspicious user",
+		defaultValue: true,
+	}),
+	declareConfig<boolean>("highlights.basic.low_trust_user", "TOGGLE", {
+		path: ["Highlights", "Built-In"],
+		label: "Show Low Trust User Highlights",
+		hint: "Whether or not to highlight users who are marked as a low trust user",
+		defaultValue: true,
+	}),
```
Comment (AnatoleAM) [inline src/site/twitch.tv/modules/chat/ChatModule.vue:367]: "Low Trust User" and "Suspicious User" are synonymous. The former is just the technical term used by Twitch. The only differentiation is "Monitored" and "Restricted".

**Review by AnatoleAM — CHANGES_REQUESTED** [review]

File: src/site/twitch.tv/modules/chat/ChatModule.vue (hunk 3 above)
Comment (Niller2005) [reply src/site/twitch.tv/modules/chat/ChatModule.vue:367]: Should both of these options be moved to "Chat > Moderation" instead and change the Restricted to opacity so it's always visible?

Comment (AnatoleAM) [reply src/site/twitch.tv/modules/chat/ChatModule.vue:367]: Keep it under highlights for now. Ideally these things should actually sync up with the native highlights menu from twitch, but we don't have hooks into this yet. I may look into it sometime if no one else does

**Review by AnatoleAM — APPROVED** [review]
This is fine for now.

I'd like to note though in the future we should look at getting this from twitch's highlight settings.

**Pull request merged**